.github/workflows/auto-translate.yml

```diff
@@ -0,0 +1,28 @@
+name: Check for Updates to Translations
+
+on:
+  push:
+    branches:
+      - 'trunk'
+  workflow_dispatch:
+
+# Cancels all previous workflow runs for the branch that have not completed.
+concurrency:
+  # The concurrency group contains the workflow name and the branch name.
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+  translate:
+    name: 'Check and update translations'
+    permissions:
+      contents: write
+      pull-requests: write
+    uses: newfold-labs/workflows/.github/workflows/reusable-translations.yml@main
+    with:
+      text_domain: 'wp-module-onboarding'
+      pr_branch_name: 'trunk'
```

Comment:
```suggestion
```

This is no longer required or accepted. The `github.ref_name` will be used by default.